yew/src/agent/link.rs

```diff
@@ -188,3 +193,34 @@ where
         }
     }
 }
+
+#[cfg(test)]
+#[cfg(feature = "agent")]
+pub(crate) mod tests {
+    use super::*;
+
+    struct TestAgent;
+    impl Agent for TestAgent {
+        type Reach = Context<Self>;
+        type Message = ();
+        type Input = ();
+        type Output = ();
+
+        fn create(_link: AgentLink<Self>) -> Self {
+            TestAgent
+        }
+
+        fn update(&mut self, _msg: Self::Message) {
+            unimplemented!();
+        }
+
+        fn handle_input(&mut self, _msg: Self::Input, _asker: HandlerId) {
+            unimplemented!();
+        }
+    }
+
+    #[test]
```

Comment:
This is not picked up by `cargo test --doc --features doc_test,wasm_test,yaml,msgpack,cbor,toml \
    --features std_web,agent,services --no-default-features` for some reason